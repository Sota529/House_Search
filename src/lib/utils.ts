export const univs = [
  { id: "Tokyo", Name: "東京" },
  { id: "Waseda", Name: "早稲田" },
  { id: "Tokyorika", Name: "東京理科" },
  { id: "Tokoha", Name: "常葉" },
  { id: "Keio", Name: "慶応" },
  { id: "Meizi", Name: "明治" },
  { id: "Chiba", Name: "千葉" },
  { id: "Rikyo", Name: "立教" },
  { id: "Hosei", Name: "法政" },
  { id: "Dentsu", Name: "電気通信" },
];
export function getPostId() {
  return univs.map(({ id }) => {
    return {
      params: {
        id: id,
      },
    };
  });
}
