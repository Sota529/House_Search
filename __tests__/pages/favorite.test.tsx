import React from "react";
import Favorite from "src/pages/favorite";
import { render } from "../utils/index";

describe("トップページ", () => {
  it("スナップショットテスト", () => {
    const { container } = render(<Favorite />);
    expect(container).toMatchSnapshot();
  });
});
