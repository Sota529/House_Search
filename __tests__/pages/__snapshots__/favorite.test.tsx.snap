// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`トップページ スナップショットテスト 1`] = `
<div>
  <h2
    class="chakra-heading css-1me5kep"
  >
    お気に入り
  </h2>
  <div
    class="css-gmuwbf"
  >
    <div
      class="chakra-spinner css-lojytg"
    >
      <span
        class="css-f8n5zr"
      >
        Loading...
      </span>
    </div>
  </div>
  <div
    class="css-1n2mv2k"
  >
    <p
      class="chakra-text css-1jijfcn"
    >
      お気に入りにした物件は
      <br />
      ありません
    </p>
  </div>
</div>
`;
